app/assets/stylesheets/application.scss

```diff
@@ -69,14 +89,47 @@ body {
   transition: all 0.2s;
 }
 
+.big-checkbox {width: 20px; height: 20px;}
+
+.btn-submit {
+  font-size: 80%;
+  border-radius: 5rem;
+  letter-spacing: .1rem;
+  font-weight: bold;
+  padding: 1rem;
+  transition: all 0.2s;
+}
+
 .form-label-group {
   position: relative;
   margin-bottom: 1rem;
+  width: 100%;
 }
 
+.form-group-search {
+  position: relative;
+  margin-bottom: 1rem;
+  width: 100%;
+  text-align: left;
+  border-radius: 2rem;
+  display: block;
+  width: 100%;
+  margin-bottom: 0;
+  line-height: 1.5;
+  padding: 0.5rem;
+}
+
+.table-striped>tbody>tr:nth-child(odd)>td,
+.table-striped>tbody>tr:nth-child(odd)>th {
+   background-color: #edeeea; // Choose your own color here
+ }
```

Comment:
customizes the color of the alternating stripes on the table.

Reply:
Cool. I can't wait to check this out.

Reply:
Nice Work!